codespell_lib/data/dictionary.txt

```diff
@@ -16875,6 +16989,7 @@ exmaines->examines
 exmaple->example
 exmaples->examples
 exmple->example
+exmples->examples
```

Comment:
Adverb too?
```exmplary->exemplary```

Reply:
I added `exemplar(s)`, `exemplary`, `exemplify`+friends.
`exemplarily` `exemplariness` `exemplarity` seem too obscure to be useful.